go.sum

```diff
@@ -254,9 +254,14 @@ k8s.io/utils v0.0.0-20190607212802-c55fbcfc754a h1:2jUDc9gJja832Ftp+QbDV0tVhQHMI
 k8s.io/utils v0.0.0-20190607212802-c55fbcfc754a/go.mod h1:sZAwmy6armz5eXlNoLmJcl4F1QuKu7sr+mFQ0byX7Ew=
 sigs.k8s.io/cluster-api v0.0.0-20190809134526-b8af96f0a42a h1:aDiczQxkpWlxe5yzqm6rJNe5orMuHTt7Q0MqDJOJ/bg=
 sigs.k8s.io/cluster-api v0.0.0-20190809134526-b8af96f0a42a/go.mod h1:CgxuxJy6W8onO8duP2ED+s0kwNxsNWIGEXMnSUsxGcw=
+sigs.k8s.io/cluster-api v0.0.0-20190820130448-9fd8e4cbea0f h1:gL8HQOSMGhMvL2BZNQSuKdXe02se34Uf29Rvq3iOfP8=
+sigs.k8s.io/cluster-api v0.0.0-20190820130448-9fd8e4cbea0f/go.mod h1:zBOsRI6nVKzD6CPmA/3qYpLCckc5FVxMBxhvBxuDNsg=
```

Comment:
We should run `go mod tidy` or look at why we're getting double entries for these modules

Reply:
I'm trying to work through issues now, but `go mod` is making me angry, `go mod tidy` tries to switch to capi v0.1.9 currently.

Reply:
We should have a `replace` directive for Cluster API, k8s.io/api and apimachinery, like in CAPA:

```
replace (
	k8s.io/api => k8s.io/api v0.0.0-20190704095032-f4ca3d3bdf1d
	k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20190704094733-8f6ac2502e51
	sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v0.0.0-20190819183132-38edf8b4497e
)
```